edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_learner-profile.scss

```diff
@@ -0,0 +1,441 @@
+// Style for the profile template
+
+$profile-background: #FFFFFF;
+$profile-secondary-background: #FAFAFA;
+$profile-border-color: #CCCCCC;
+$font-size-small: 0.875em;
+
+@mixin button($background: $primary, $border: 1px solid $primary, $color: #FFF, $background-active: #991f34, $color-active: #FFF, $border-color-active: #991f34) {
```

Comment:
learner-profile-button @alexjmpb?
Line too long, split out these parameters.
You already have a variable definition for some of these colours, please use the variables instead of hardcoding the values.